### DISCUSSION TITLE
Atlas Scientific  ORP-- Consumer versus Industrial Grade

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#708] [REM] [answered]
Ordering equipment for a chem controller.

On the wiki, it looks like you are using Industrial grade PH and ORP probes.  It looks like I could buy 4 consumer grade ORP probes for the price of one Industrial grade.

The connectors are different and mounting the consumer grade probes on a flow cell would require some mechanical design.  Any thing else?

Why did you go with the Industrial grade?

**Comment by rstrouse** [comment, marked as answer]
Actually there is an adapter for the consumer and lab grade probes for both 3/4 and 1/2 holes.  The reason for me were that there were quite a few failures early on with the lab and consumer grade probes where they would go out of calibration.  For me, the industrial probes work very well and I calibrate once a year whether I need it or not.

**Reply by davthomaspilot** [reply]
I plan to use the probes that are in the Intellichem flow cell currently, until they stop working.  

I've had to replace them twice in seven or eight years and they were expensive.  

I have to clean them once or twice a year anyway and it's really easy to change them.   So, for me, it's all about what's the lowest cost over a five or ten year period.

**Reply by rstrouse** [reply]
I have not tried it but the IntelliChem probes should work with the EZO circuits.  This will give you a 3 point calibration curve.
